README.md

```diff
@@ -193,6 +193,24 @@ An example for content negotiation based on HTTP headers and with query paramete
    accept: application/json
    => should return the response using the GML encoding
 
+#### Content negotiation for coordinate reference systems (CRS)
+Request and response may be based on another coordinate reference system. This applies the HTTP-mechanism for content negotiation. The CRS of the geometry in the request (request body) is specified using the header `Content-Crs`. Below are examples for four coordinate reference systems any other CRS can be supported in the same manner.
+
+|HTTP header|Value|Explanation|
+|-|-|-|
+|`Content-Crs`|EPSG:4326|WGS84, global|
```

Comment:
The [draft OGC spec](https://github.com/opengeospatial/conneg-by-crs/blob/master/18-072-Content-Negotiation-by-CRS.md) requires URIs a values. I.e., the value should be `http://www.opengis.net/def/crs/EPSG/0/4326`, not `EPSG:4326`.

Reply:
It should also be added that this is work-in-progress and neither an approved standard in IETF / OGC nor widely implemented. There should not be an implication that at this time this is the preferred or only way to request / state the CRS of content.